opensearch-operator/pkg/builders/dashboards_test.go

```diff
@@ -0,0 +1,33 @@
+package builders
+
+import (
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	opsterv1 "opensearch.opster.io/api/v1"
+)
+
+var _ = Describe("Builders", func() {
```

Comment:
This also needs a similar test for the labels, so that everything is covered.

Reply:
Test added as requested.